firefox-ios/Client/Frontend/PasswordManagement/PasswordManagerViewModel.swift

```diff
@@ -75,22 +75,22 @@ final class PasswordManagerViewModel {
     /// Searches SQLite database for logins that match query.
     /// Wraps the SQLiteLogins method to allow us to cancel it from our end.
     func queryLogins(_ query: String, completion: @escaping (([LoginRecord]) -> Void)) {
-        profile.logins.searchLoginsWithQuery(query).upon { result in
+        profile.logins.searchLoginsWithQuery(query, completionHandler: { result in
             ensureMainThread {
                 // Check any failure, Ex. database is closed
```

Comment:
can you remove this comment?

Reply:
Hi @dataports, I just removed the comment from line 80 and pushed the updated code. Thanks 🙏